pkg/services/govmomi/clustermodules/provider.go

```diff
@@ -88,20 +91,19 @@ func (cm *provider) DoesModuleExist(ctx context.Context, moduleUUID string, clus
 		return false, nil
 	}
 
-	modules, err := cm.manager.ListModules(ctx)
-	if err != nil {
-		return false, err
+	_, err := cm.manager.ListModuleMembers(ctx, moduleUUID)
+	if err == nil {
+		log.V(4).Info("Cluster module exists", "moduleUUID", moduleUUID)
```

Comment:
```suggestion
		log.V(4).Info("Cluster module exists")
```

moduleUUID is already added a few levels above. If we add it here we will have a duplicate k/v pair.

I know it's not ideal that we have to care about that, but that's how klog behaves (for performance reasons, they don't want to go through k/v pair deduplication for every log line)

Reply:
ouch, had no idea :) removing it here

Reply:
btw I have also removed the clusterRef from the logging, as we don't use it anymore on any query

Reply:
Sounds good!